spec/support/rubygems_ext.rb

```diff
@@ -23,7 +23,8 @@ module Rubygems
       "rake" => "13.0.1",
       "builder" => "~> 3.2",
       # ruby-graphviz is used by the viz tests
-      "ruby-graphviz" => ">= 0.a",
+      # for >= Ruby 2.3
+      "ruby-graphviz" => "1.2.4",
```

Comment:
Thanks for fixing this! I made a few changes to ruby-graphviz to support ruby 2.8 but unintentionally dropped support for ruby 2.3, so this are a bit complicated now. I will follow up with a PR to fix ruby-head build by using the new `ruby-graphviz 1.2.5` version on ruby-head.